packages/super-components/components/msc-tile/docs/msc-tile.stories.ts

```diff
@@ -22,17 +22,62 @@ export default {
     </section>
   `,
   argTypes: {
-    tileNumber: { control: 'number', default: 1 },
-    tileType: { control: 'text' },
-    tileTitle: { control: 'text' },
-    tileDescription: { control: 'text' },
-    href: { control: 'text' },
-    isExternalHref: { control: 'boolean' },
-    seeMoreLabel: { control: 'text' },
-    imgSrc: { control: 'text' },
-    imgAlt: { control: 'text' },
-    hasBadges: { control: 'boolean' },
-    hasFooter: { control: 'boolean' },
+    tileNumber: {
+      control: 'number',
+      min: 1,
+      default: 1,
+      description: 'Number of tiles to display (only in the storybook)',
+    },
+    tileType: {
+      description: 'Change the top label of the tile according to its type',
+      control: 'select',
+      options: ['product', 'faq'],
+    },
+    tileTitle: { control: 'text', description: 'Title of the tile' },
+    tileDescription: { control: 'text', description: 'Texte of the tile' },
```

Comment:
```suggestion
    tileDescription: { control: 'text', description: 'Description of the tile' },
```

Reply:
Fixed on this branch : https://github.com/ovh/manager/pull/9986